bitnami/mongodb/templates/arbiter/statefulset.yaml

```diff
@@ -54,9 +54,64 @@ spec:
       {{- if .Values.arbiter.podSecurityContext.enabled }}
       securityContext: {{- omit .Values.arbiter.podSecurityContext "enabled" | toYaml | nindent 8 }}
       {{- end }}
-      {{- if .Values.arbiter.initContainers }}
-      initContainers: {{- include "common.tplvalues.render" (dict "value" .Values.arbiter.initContainers "context" $) | nindent 8 }}
-      {{- end }}
+      initContainers: 
```

Comment:
In case you don't set the `arbitrer.initContainer` nor the `tls.enabled` this will render empty. will it fail?

Reply:
Nope, I Just tested it. The chart will indeed install if arbiter.initContainer and/or tls.enabled is not set.